controllers/glance_controller.go

```diff
@@ -529,7 +529,7 @@ func (r *GlanceReconciler) reconcileNormal(ctx context.Context, instance *glance
 	//
 
 	serviceLabels := map[string]string{
-		common.AppSelector: glance.ServiceName,
+		common.AppSelector: instance.Name,
```

Comment:
We don't support multiple `Glances` (the top level resource) in the same namespace, hence -1 here. 
`glance.ServiceName` looks correct as it points to the associated const [1]

[1] https://github.com/openstack-k8s-operators/glance-operator/blob/main/pkg/glance/const.go#L24

Reply:
The main concern behind this is that we are naming resources inconsistently. I agree we do not support multiple `Glance`s in the same name space. Then why do we name some resources such as database or config map according to `instance.Name` ?

I've spent more time looking into the other operators and I later noticed there are some logics not accepting multiple keystoneapi resources in a single namespace... then probably a better solution here is to implement the validation to avoid deployment of multiple CRs in a single namespace(unless that is required), and use the static names consistently across the all resources.